brevbaker-api-model-common/api/brevbaker-api-model-common.api

```diff
@@ -334,6 +334,7 @@ public abstract interface class no/nav/pensjon/brevbaker/api/model/LetterMarkup
 	public abstract fun getSakspart ()Lno/nav/pensjon/brevbaker/api/model/LetterMarkup$Sakspart;
 	public abstract fun getSignatur ()Lno/nav/pensjon/brevbaker/api/model/LetterMarkup$Signatur;
 	public abstract fun getTitle ()Ljava/lang/String;
+	public fun validate ()V
```

Comment:
Trur eg synes det gir meining at valideringsmetodane ligg i API-et, så vi synleggjer for konsumentar at det er naturleg å validere objektet/objekta, og at vi tilbyr ei fornuftig standard-validering, men at du sjølv må kalle ho.